src/lib/core/components/computations/getAppsDefaultConfigs.tsx

```diff
@@ -0,0 +1,40 @@
+import { useStudyMetadata } from '../..';
+import { useCollectionVariables } from '../../hooks/study';
+import { ComputationAppOverview } from '../../types/visualization';
+
+const defaultValues = {
+  abundance: {
+    name: 'RankedAbundanceComputation',
+    method: 'median',
+    methodPropertyName: 'rankingMethod',
+  },
+  alphadiv: {
+    name: 'AlphaDivComputation',
+    method: 'shannon',
+    methodPropertyName: 'alphaDivMethod',
+  },
+};
+
+export const useDefaultPluginConfiguration = (
+  apps: ComputationAppOverview[]
+) => {
+  const studyMetadata = useStudyMetadata();
+  const collections = useCollectionVariables(studyMetadata.rootEntity);
+  return apps.map((app) => {
+    // @ts-ignore
+    const defaultObject = defaultValues[app.name];
+    if (!defaultObject) return null;
+    return {
+      name: app.name,
+      configuration: {
+        name: defaultObject.name,
+        collectionVariable: {
+          variableId: collections[0].id,
+          entityId: collections[0].entityId,
+        },
+        [defaultObject.methodPropertyName]: defaultObject.method,
+        description: `Data: ${collections[0].entityDisplayName}: ${collections[0].displayName}; Method: ${defaultObject.method}`,
```

Comment:
You might already know this, but I think this is the source of the 422. I think this should be moved to a `displayName` prop in the top-level object (sibling of `name` and `configuration`).